src/base/static/components/app.tsx

```diff
@@ -263,10 +264,14 @@ class App extends Component<Props, State> {
     resolvedConfig.right_sidebar.is_visible_default &&
       this.props.updateUIVisibility("rightSidebar", true);
 
+    // TODO:
     languageModule.changeLanguage(Mapseed.languageCode);
 
     // The config and user data are now loaded.
     this.setState({
+      currentLanguage: "es",
```

Comment:
Temporarily hard-cording `es` here; eventually this value will be updated by the language menu.